src/Cli/dotnet/ToolPackage/ToolPackageDownloader.cs

```diff
@@ -193,6 +205,32 @@ internal class ToolPackageDownloader : IToolPackageDownloader
             lockFileLib.ToolsAssemblies.AddRange(toolsGroup);
         }
 
+        private static void UpdateRuntimeConfig(
+            ToolPackageInstance toolPackageInstance
+            )
+        {
+            for (int i = 0; i < toolPackageInstance.Commands.Count; i++)
+            {
+                var command = toolPackageInstance.Commands[i];
```

Comment:
Nit: Looks like this can be simplified with a `foreach` loop.

```suggestion
            foreach (var command in toolPackageInstance.Commands)
            {
```